### PR TITLE
Fixed the scrolling issue occurring in mobile

### LINKED ISSUE #368: Tab bar prevents swipe up/down on mobile browser
Hi, actually I'm coming here because of an issue that I've found in ant-design's tab. I recently discovered that the location of the problem was in `rc-tabs`. So I tried cloning the master branch and inspecting where the problem was. Finally I found that this problem can be solved by removing `e.preventDefault()` in `useTouchMove.ts` file. 

You can see it on the branch I forked:
[https://github.com/irvanherz/tabs/commit/251d8713e6751650f52585d83aa9349c9f7a13c8#diff-6098251a64dc37073e837fba36078438a119c06311050fdbc9975bcbd632c60e](https://github.com/irvanherz/tabs/commit/251d8713e6751650f52585d83aa9349c9f7a13c8#diff-6098251a64dc37073e837fba36078438a119c06311050fdbc9975bcbd632c60e)

Is this small change ok to fix the bug?
### Reproduction link
[Edit on CodeSandbox](https://codesandbox.io/s/antd-reproduction-template-forked-w7i44)

### Steps to reproduce
From any mobile browser, try to swipe view up or down by starting finger touching **Tabs' bar**.

### What is expected?
It should scrolling view up/down as I swipe.

### What is actually happening?
It does nothing.

| Environment | Info |
|---|---|
| antd | 4.13.1 |
| React | 17.0.1 |
| System | Android 7.1.2; Redmi 5A Build/N2G47H |
| Browser | Chrome Version 88.0.4324.181 |

---
My project using many tabs at home page. This bug oftenly brings problem for any users who open the page from mobile browser.

### PR DESCRIPTION
**Pull request by pplcallmesatz**
We tried to scroll from the particular element in mobile it preventing to scroll. When we removed the `e.preventDefault(); ` working fine for Us. 

Thanks!

resolve #368

### CODE REVIEW

File: src/hooks/useTouchMove.ts [hunk 1]
```diff
@@ -30,8 +30,6 @@ export default function useTouchMove(
 
   function onTouchMove(e: TouchEvent) {
     if (!touchPosition) return;
-
-    e.preventDefault();
```
Comment (zombieJ) [inline src/hooks/useTouchMove.ts:34]: Should prevent by move direct instead. Not remove directly

Comment (Chorer) [reply src/hooks/useTouchMove.ts:34]: > Should prevent by move direct instead. Not remove directly

What “prevent by move direct ” means?

Comment (zombieJ) [reply src/hooks/useTouchMove.ts:34]: For horizontal preventDefault with up & down but not block with left & right. And vertical is prevent up & down but not left & right.

Comment (Chorer) [reply src/hooks/useTouchMove.ts:34]: > For horizontal preventDefault with up & down but not block with left & right. And vertical is prevent up & down but not left & right.

It seems @pplcallmesatz didn't make some new commits. Why can't you just fix the problem?

**Comment by afc163** [comment]
Fixed in https://github.com/react-component/tabs/pull/622

**Pull request closed without merging**